app.go

```
@@ -211,16 +227,16 @@ func (a *App) Run(arguments []string) (err error) {
 	}
 
 	if err != nil {
-		if a.OnUsageError != nil {
-			err := a.OnUsageError(context, err, false)
+		if onUsageError := a.resolveOnUsageError(); onUsageError != nil {
+			err := onUsageError(context, err, false)
 			HandleExitCoder(err)
 			return err
 		}
 		fmt.Fprintf(a.Writer, "%s %s\n\n", "Incorrect Usage.", err.Error())
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)